api/apiFunctions.ts

```diff
@@ -249,10 +261,19 @@ export async function createWeeklyPicks({
   userResults,
 }: IWeeklyPicks): Promise<Models.Document> {
   try {
+    const getLeagueGameResults = await databases.listDocuments(
+      appwriteConfig.databaseId,
+      Collection.GAME_RESULTS,
+      [
+        Query.equal('leagueId', leagueId),
+        Query.equal('gameWeekId', gameWeekId),
+      ],
+    );
+
     return await databases.updateDocument(
       appwriteConfig.databaseId,
       Collection.GAME_RESULTS, //collectionID
-      '663130a100297f77c3c8', //documentID
+      getLeagueGameResults.documents[0].$id, //documentID
```

Comment:
Are you sure this is always going to be Index 0?

Reply:
Shashi beat me to the punch here. 

I'm assuming this will always be an index of 0 bc There should only be one `userResults` per week, right?

Reply:
Yes, I am sure.  There should only be one document $id for that week.  which would be [0] in the returned array.  If we were looking at MULTIPLE weeks then it would not be zero.